### PR TITLE
Fix fill names and add test to check for `x` and `y`

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Align `c_fill` to receive x and y qualifiers and add test to it.

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [ ] New feature
- [x] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt [hunk 1]
```diff
@@ -25,30 +25,30 @@ class Fill(
 
     class Builder : BaseBuilder<Builder>() {
         private var gravity: Gravity? = null
-        private var offsetX: Any? = null
-        private var offsetY: Any? = null
+        private var x: Any? = null
```
Comment (patrick-tolosa) [inline url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt:28]: It's worth adding a comment that x,y are only working with `g_xy_center`.

The test should also reflect that

**Review by patrick-tolosa — CHANGES_REQUESTED** [review]

File: url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt [hunk 2]
```diff
@@ -25,30 +25,30 @@ class Fill(
...
             this.gravity = gravity
         }
 
-        private fun offsetX(x: Any) = apply {
-            this.offsetX = x
+        private fun x(x: Any) = apply {
+            this.x = x
         }
 
-        fun offsetX(x: String) = offsetX(x as Any)
```
Comment (const-cloudinary) [inline url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt:39]: Old functions should be deprecated and new one should be added

File: url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt (hunk 1 above)
Comment (const-cloudinary) [reply url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt:28]: @patrick-tolosa , x and y are also supported for fill and other crop modes as well

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
See my comments

**Review by const-cloudinary — APPROVED** [review]

File: url-gen/src/test/kotlin/com/cloudinary/transformation/ResizeTest.kt [hunk 3]
```diff
@@ -163,6 +163,10 @@ class ResizeTest {
 
     @Test
     fun testFill() {
+        cldAssert("c_fill,x_100,y_100", Resize.fill { offsetX(100)
```
Comment (patrick-tolosa) [inline url-gen/src/test/kotlin/com/cloudinary/transformation/ResizeTest.kt:166]: I'm not sure this does anything, however the test is correct.

It looks like it needs to be appended to g_xy_center

See this 
https://res.cloudinary.com/demo/image/upload/h_2400,w_1080,c_fill,g_xy_center,x_500/sample.jpg

vs this

https://res.cloudinary.com/demo/image/upload/h_2400,w_1080,c_fill,x_500/sample.jpg

**Review by patrick-tolosa — APPROVED** [review]

**Pull request merged**